src/Tribe/Tickets_Handler.php

```diff
@@ -124,6 +124,36 @@ public function print_checkedin_totals() {
 	}
 
 	/**
+	 * Get a count of all the tickets for an event. Queries all active modules/providers.
+	 *
+	 * @static
+	 *
+	 * @param $event_id
+	 *
+	 * @return int number of tickets ( 0 means unlimited )
+	 */
+	public function get_event_tickets_count() {
```

Comment:
You have `$event_id` in the docblock but it isn't reflected in the parameter list. I'd love to see `$event_id` passed in (though, call it `$post_id`) and default to the global post only if `empty( $post_id )`.